zerver/tests/test_subs.py

```diff
@@ -2761,7 +2952,7 @@ def test_realm_admin_remove_others_from_subbed_private_stream(self) -> None:
         are on.
         """
         result = self.attempt_unsubscribe_of_principal(
-            query_count=17,
+            query_count=19,
```

Comment:
We are changing the event structure to only send stream IDs instead of the full stream objects, so these queries will not be needed after that change.

Reply:
@shubham-padia can you make sure we have an issue tracking that? I think there's an API design thread about it, and maybe an issue already, but we should make sure it's linked here.

Reply:
PR at #32442